docs-source/index.md

```diff
@@ -124,10 +126,10 @@ To start using the API, you need to create an instance of the @Auth0.ManagementA
 var client = new ManagementApiClient("token", "YOUR_AUTH0_DOMAIN");
 ```
 
-> You can obtain your Auth0 Domain from the [Application section of the Auth0 Dashboard](https://manage.auth0.com/#/applications) by looking at the settings for your particular application. 
+> You can obtain your Auth0 Domain from the [Application section of the Auth0 Dashboard](https://manage.auth0.com/#/applications) in the settings for your particular application. 
 
 > [!NOTE]
-> For details on how to generate the token, please see the "Getting an API token" section of the [Management API documentation](https://auth0.com/docs/api/v2)
+> For details on how to generate the token, please see the [Access Token for the Management API](https://auth0.com/docs/api/management/v2/tokens)
```

Comment:
```suggestion
> For details on how to generate the token, please see [Access Token for the Management API](https://auth0.com/docs/api/management/v2/tokens)
```

I'd suggest linking to a specific page here, maybe this one?

https://auth0.com/docs/api/management/v2/get-access-tokens-for-test

I'm not sure what the surrounding documentation is here but this is a catching point for a lot of folks new to the Management API. They might think they should generate a token with a really long expiration and just change it out regularly. If it's not there, I'd mention that the manual process is for testing and the CC grant is for production:

https://auth0.com/docs/api/management/v2/get-access-tokens-for-production

Reply:
I think the page currently being linked to is a good summary and includes both the links you mention on it.